bookkeeper-server/src/main/java/org/apache/bookkeeper/client/AsyncCallback.java

```diff
@@ -17,6 +17,7 @@
 package org.apache.bookkeeper.client;
 
 import java.util.Enumeration;
+import org.apache.bookkeeper.client.api.LedgerEntry;
```

Comment:
Oh I think there is a problem here - this is the legacy api, you can't change this to `api.LedgerEntry`. try remove this line and recompile, you will find it will not compile. I think `client.LedgerEntry` should extend `impl.LedgerEntryImpl`. The current approach for `client.LedgerEntry` doesn't really work.

Reply:
I C. need changes